docs/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
```

Comment:
1) Can the Makefile stuff not be added to our main Makefile?

Otherwise like Makefile is:

> No newline at end of file